lib/src/presentation/view/upcoming_movies.dart

```diff
@@ -0,0 +1,73 @@
+import 'package:flutter/material.dart';
+
+import '../../core/state/state.dart';
+import '../../core/util/ui_constants.dart';
+import '../../domain/entity/movie.dart';
+import '../bloc/i_bloc_genres.dart';
+import '../bloc/i_bloc_upcoming.dart';
+import '../widget/back_button.dart';
+import '../widget/list_grid_upcoming.dart';
+
+class UpcomingMovies extends StatefulWidget {
+  const UpcomingMovies({
+    super.key,
+    required this.blocUpcoming,
+    required this.blocGenres,
+  });
+
+  final IBlocUpcoming blocUpcoming;
+  final IBlocGenres blocGenres;
+
+  @override
+  State<UpcomingMovies> createState() => _UpcomingMoviesState();
+}
+
+class _UpcomingMoviesState extends State<UpcomingMovies> {
+  late List<Movie> movieList;
+
+  @override
+  void initState() {
+    super.initState();
+    widget.blocUpcoming.getUpcoming();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      floatingActionButton: const CustomBackButton(),
+      floatingActionButtonLocation: FloatingActionButtonLocation.startDocked,
+      body: StreamBuilder<DataState<List<Movie>>>(
+        stream: widget.blocUpcoming.movies,
+        initialData: DataState(
+          state: ActualState.loading,
+        ),
+        builder: (context, snapshot) {
```

Comment:
Same as above and you missed a trailing comma

Reply:
done